pandas/core/frame.py

```diff
@@ -2326,7 +2326,7 @@ def _sizeof_fmt(num, size_qualifier):
             else:
                 _verbose_repr()
 
-        counts = self.get_dtype_counts()
+        counts = self._data.get_dtype_counts()
```

Comment:
this one?

Reply:
I think this is okay. It's internal usage and slightly more performant I would think than `dtype.value_counts()` (left as a dictionary as opposed to constructing the Series)

Reply:
can you remove get_dtype_counts() from blocks its unecessary as well

Reply:
Looks to be needed to get the dtypes later on for `info`?